app/settings/routes.py

```diff
@@ -8,6 +8,9 @@
 @bp.route('/preferences', methods=['GET', 'POST'])
 @login_required
 def user_preferences():
+    """
+    Updates the user preferences
+       """
```

Comment:
In the future, please pay more attention to your formatting. Miss alignments in python are usually a big deal.